src/client/app/helpers/strings.ts

```diff
@@ -112,6 +112,8 @@ export class Strings {
 
     static readonly deleteSnippetConfirm = 'Are you sure you want to delete this snippet?';
 
+    static readonly triggerActionLabel = 'Trigger Suggest';
```

Comment:
Now that I actually remember what it does I would think either 'editorTriggerSuggestMenuItemLabel' or 'editorTriggerSuggestContextMenuLabel'